devito/ops/types.py

```diff
@@ -0,0 +1,15 @@
+import devito.types.basic as basic
+
+__all__ = ['String']
+
+
+class String(basic.Basic):
```

Comment:
ops_timing_output receives an empty string. When we passing Call(namespace['ops_timing_output'], [**' '**]), an error is thrown, because str() does not have a value.

Reply:
And File ".../devito/ir/iet/visitors.py", line 169, in _args_call
    if i.is_LocalObject:
AttributeError: 'str' object has no attribute 'is_LocalObject'

Reply:
ok